### PR TITLE
State fixes and improvements

### PR DESCRIPTION
**Pull request by zer0main**

### CODE REVIEW

File: pkg/storage/accountsstorage_test.go [hunk 1]
```diff
@@ -15,37 +15,21 @@ const (
 	TOTAL_BLOCKS_NUMBER = 200
```
Comment (alexeykiselev) [inline pkg/storage/accountsstorage_test.go]: Please use **camelCase** or **UpperCamelCase** for constants.

File: pkg/importer/importer.go [hunk 2]
```diff
@@ -15,19 +15,20 @@ import (
...
 }
 
-func ApplyFromFile(st State, blockchainPath string, nBlocks int, checkBlocks bool) error {
+func ApplyFromFile(st State, blockchainPath string, nBlocks, startHeight uint64, checkBlocks bool) error {
 	blockchain, err := os.Open(blockchainPath)
 	if err != nil {
 		return errors.Errorf("failed to open blockchain file: %v\n", err)
 	}
 	sb := make([]byte, 4)
 	buf := make([]byte, 2*1024*1024)
```
Comment (alexeykiselev) [inline pkg/importer/importer.go]: Better create not slice of bytes, but fixed byte array of the maximum block size. And later create a slice of required length from it.

**Comment by zer0main** [comment]
> I think that `keys.go` should be not in `proto` package but in `state` package.

The problem is that `proto` package uses keys too, and `state` already imports `proto`.

File: pkg/importer/importer.go (hunk 2 above)
Comment (zer0main) [reply pkg/importer/importer.go]: Done.

**Pull request merged**